gaskij/rpicampusmap
# Add favicon and dynamic browser titles

## Description
There is currently no favicon for the webapp, and the title always says "RPI CampusMap" regardless of what page you are on. With these changes, a favicon is added and you can configure the browser title based on the currently rendered component.

## Solution
- Added a favicon line to `webpack.config.js` and to `index.html`
- Added the `react-helmet` package to be able to edit html `<head>` tags, like `<title>`

## Known Issues
The favicon still does not show when running the app locally, but should show when hosted

## Testing Procedures
* Navigate to `localhost:3000/favicon.ico` and ensure the favicon shows up
* Ensure the favicon shows up in the browser tab
* Navigate to the search results page and check that the browser title changed.
* Add a query on the search url, like `?query=jec`, and check that the browser title changed.

## Checklist for author
